js/main.js

```diff
@@ -71,6 +95,6 @@ addBtn.addEventListener('click', (e) => {
   document.getElementById('author').value = '';
 });
 
-window.addEventListener('DOMContentLoaded', () => {
-  DisplayBooks.display();
-});
+// window.addEventListener('DOMContentLoaded', () => {
+//   DisplayBooks.display();
+// });
```

Comment:
- [ ] You have this commented piece of code in your main.js file :fearful: Kindly remove it to make your codebase clean :+1: to follow JavaScript best practices :blush: